packages/osx_bundle/Contents/Info.plist

```diff
@@ -143,6 +143,7 @@
             <string>None</string>
 		</dict>
 	</array>
-
+  <key>NSSupportsAutomaticGraphicsSwitching</key>
```

Comment:
Does this actually work correctly? We never manually reinit the context and expect to have some persistent state.

Reply:
Reading https://developer.apple.com/library/content/technotes/tn2229/_index.html tells me we need to handle capability changes, and doesn't indicate anything else we need to do in that case. I thought we were fine since `DetectOpenGLCapabilities` runs every frame, but looking closer, we do need to update it, since it currently skips out if `maxTextureSize` is already set.